dev-util/idea-ultimate/idea-ultimate-15.0.2.143.1015.7_pre.ebuild

```diff
@@ -73,7 +73,9 @@ src_install() {
 	doins -r *
 	fperms 755 "${dir}"/bin/{idea.sh,fsnotifier{,64}}
 
-	make_wrapper "${PN}" "${dir}/bin/${MY_PN}.sh"
+	make_wrapper "${PN}" "${dir}/bin/${MY_PN}.sh" || die
+	newicon "bin/${MY_PN}.png" "${PN}.png" || die
+	make_desktop_entry "${PN}" "IntelliJ Idea Ultimate" "${PN}" "Development;IDE;" || die
```

Comment:
I reviewed your reviews and could find it either.

The rules are simple:
* all helper function (```econf```, ```emake```, ```doins``` ...) die alone
* all eclass function in EAPI >=5 die alone e.g. ```make_desktop_entry``` from eutils.eclass
* all system commands **must** have a die e.g. ```mkdir```, ```rm```, ```cd```, ```chmod```, ```ln``` ...